src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,488 @@
+   }
+) }}
+
+### Allowed 2D coordinate systems
+
+If electrodes are localized in 2D space (only x and y are specified and z is `"n/a"`), then the positions in this file MUST correspond to the locations expressed in pixels on the photo/drawing/rendering of the electrodes on the brain.
+In this case, `ephysCoordinateSystem` MUST be defined as `"Pixels"`, and `ephysCoordinateUnits` MUST be defined as `"pixels"` (note the difference in capitalization).
+Furthermore, the coordinates MUST be (row,column) pairs, with (0,0) corresponding to the upper left pixel and (N,0) corresponding to the lower left pixel.
+
+
+### Photos of the electrode positions (`*_photo.jpg`)
+
+These can include photos of the electrodes on the brain surface, photos of anatomical features or landmarks (such as sulcal structure), and fiducials. Photos can also include an X-ray picture, a flatbed scan of a schematic drawing made during surgery, or screenshots of a brain rendering with electrode positions. The photos may need to be cropped and/or blurred to conceal identifying features or entirely omitted prior to sharing, depending on obtained consent.
+
+If there are photos of the electrodes, the [acq-<label>](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#acq) entity should be specified with:
+
+* `*_photo.jpg` in case of an operative photo
+* `*_acq-xray#_photo.<extension>` in case of an x-ray picture
+* `*_acq-drawing#_photo.<extension>` in case of a drawing or sketch of electrode placements
+* `*_acq-render#_photo.<extension>` in case of a rendering
+
+The file `<extension>` for photos MUST be either `.jpg`, `.png` or `.tif`.
+
+The[ ses-<label>](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#ses) entity may be used to specify when the photo was taken.
+
+## **Multiple coordinate systems**
+
+The optional[ space-<label>](https://bids-specification.readthedocs.io/en/stable/appendices/entities.html#space) entity (`*[_space-<label>]_coordsystem.json`) can be used to indicate how to interpret the electrode positions. The space `<label>` MUST be taken from one of the modality specific lists in the[ Coordinate Systems Appendix](https://bids-specification.readthedocs.io/en/stable/appendices/coordinate-systems.html). For example for iEEG data, the restricted keywords listed under[ iEEG Specific Coordinate Systems](https://bids-specification.readthedocs.io/en/stable/appendices/coordinate-systems.html#ieeg-specific-coordinate-systems) are acceptable for `<label>`.
+
+
+## Probes (*_probes.tsv)
+
+Probes are physical devices used for recording animal ephys data. They can be permanently implanted (chronic recordings) or inserted just for the recording (acute recordings). The probe positions and properties are stored in a .tsv file. This file contains the probe id, the type of recording (acute/chronic) and the probe coordinates.
+
+{{ MACROS___make_columns_table("microephys.microephysProbes") }}
+
+
+Example of `_probes.tsv`:
```

Comment:
note, for TSVs there is ongoing
- #1794 which would render them wonderfully

you just take care about 
- annotate as `Text` for now until that PR is merged
- correctly entering them with tabs so we do not need to re-enter late

Reply:
Thanks a lot I would do that